nmdc_server/crud.py

```diff
@@ -201,12 +201,12 @@ def list_omics_processing_data_objects(db: Session, id: str) -> Query:
 
 
 def list_ko_terms_for_module(db: Session, module: str) -> List[str]:
-    q = db.query(models.KoTermToModule.term).filter(models.KoTermToModule.module == module)
+    q = db.query(models.KoTermToModule.term).filter(models.KoTermToModule.module.ilike(module))
     return [row[0] for row in q]
 
 
 def list_ko_terms_for_pathway(db: Session, pathway: str) -> List[str]:
-    q = db.query(models.KoTermToPathway.term).filter(models.KoTermToPathway.pathway == pathway)
+    q = db.query(models.KoTermToPathway.term).filter(models.KoTermToPathway.pathway.ilike(pathway))
```

Comment:
These two are only returning the "term" column, is it OK not to be able to distinguish which resulting term corresponds to which module?

Is this a real use case, where someone knows some module / pathway number, but only wants to use a prefix of it?

Reply:
Sorry I just realized you weren't using `%` here... this is just for case insensitivity then?

Reply:
I don't understand the question.  This is to fix no results returned when I search for `M00000` instead of `m00000`.  There are no wildcards in the query, so I don't' understand the prefix question.